userguide/tutorials/aws-multitier.adoc

```diff
@@ -431,6 +431,7 @@ The `diagnostic` option of the `kpm` command creates an extensive report for a g
   --killbill-api-credentials=KEY SECRET \
   --kaui-web-path=/var/lib/tomcat/webapps2 \
   --killbill-url=http://127.0.0.1:8080 \
+  --account-export=ACCOUNT_ID \
   --database-host=DBURL:3306
 ```
```

Comment:
Could you revert this change? As [discussed](https://github.com/killbill/killbill-docs/pull/464#issuecomment-1418455253), @jdmooney would be making this change as part of his work.

Reply:
Hi @reshmabidikar removed  the account export line.